config/crd/bases/infrastructure.cluster.x-k8s.io_ibmvpcmachines.yaml

```diff
@@ -245,11 +245,22 @@ spec:
             description: IBMVPCMachineSpec defines the desired state of IBMVPCMachine.
             properties:
               image:
-                description: 'Image is the id of OS image which would be install on
-                  the instance. Example: r134-ed3f775f-ad7e-4e37-ae62-7199b4988b00
-                  TODO: allow user to specify a image name is much reasonable. Example:
-                  ibm-ubuntu-18-04-1-minimal-amd64-2'
+                description: 'Deprecated: Use ImageRef instead Image is the id of
```

Comment:
small nit: this statement is not clear, probably should be: `'Deprecated: Use ImageRef instead. Image is the id of ...`

Reply:
Fixed.